global_config/domain.sh

```diff
@@ -0,0 +1,10 @@
+CONFIG=sandbox
+CONFIG_SHORT=sb
+AZURE_SUBSCRIPTION=s189-teacher-services-cloud-production
+RESOURCE_NAME_PREFIX=s189p01
+ENV_TAG=Sandbox
```

Comment:
if required, then CONFIG, CONFIG_SHORT and ENV_TAG would reference production I would think.